### DISCUSSION TITLE
Addiitional Bitmaps

### DISCUSSION THREAD

**Discussion by Marcusv2** [piotrwawrzyn/simple-arduino-crypto-display#6] [General]
Thanks for the work on this.

I thought I'd share a couple of bitmaps for those wanting these coins

Polka Dot:
  {"polkadot",
  {0x00, 0x00, 0x00, 0x00, 0x00, 0xC0, 0x01, 0x00, 0x00, 0xF8, 0x1F, 0x00, 
  0x00, 0xFE, 0x7F, 0x00, 0x00, 0xBF, 0xFF, 0x00, 0x80, 0x0F, 0xF0, 0x01, 
  0xC0, 0x07, 0xE0, 0x01, 0xE0, 0x01, 0xC0, 0x03, 0xE0, 0x01, 0x80, 0x07, 
  0xE0, 0x00, 0x00, 0x07, 0xF0, 0x00, 0x00, 0x07, 0xF0, 0x20, 0x00, 0x07, 
  0x70, 0x30, 0x00, 0x07, 0x70, 0x78, 0x00, 0x0F, 0x70, 0x78, 0x00, 0x07, 
  0xF0, 0x38, 0x00, 0x07, 0xE0, 0x38, 0x00, 0x07, 0x60, 0x3C, 0x80, 0x07, 
  0x00, 0x1C, 0xC0, 0x03, 0x00, 0x1C, 0xE0, 0x01, 0x00, 0x1C, 0xF8, 0x00, 
  0x00, 0xDC, 0xFF, 0x00, 0x00, 0xFE, 0x3F, 0x00, 0x00, 0xFE, 0x0F, 0x00, 
  0x00, 0x7E, 0x01, 0x00, 0x00, 0x1E, 0x00, 0x00, 0x00, 0x0F, 0x00, 0x00, 
  0x00, 0x0F, 0x00, 0x00, 0x00, 0x07, 0x01, 0x00, 0x00, 0xC7, 0x03, 0x00, 
  0x00, 0xC7, 0x03, 0x00, 0x00, 0x82, 0x01, 0x00,}}};

DogeCoin:
 {"dogecoin",
  {0x00, 0x00, 0x00, 0x00, 0x00, 0x0C, 0x00, 0x00, 0x00, 0x14, 0x00, 0x06, 
  0x00, 0x24, 0x00, 0x05, 0x00, 0x24, 0x80, 0x04, 0x00, 0x44, 0x40, 0x00, 
  0x00, 0x84, 0x3F, 0x04, 0x00, 0x02, 0x00, 0x04, 0x80, 0x01, 0x00, 0x04, 
  0x60, 0x00, 0x00, 0x04, 0x20, 0x00, 0x00, 0x04, 0x10, 0x01, 0x00, 0x08, 
  0x88, 0x03, 0x01, 0x10, 0x08, 0x81, 0x07, 0x10, 0x08, 0x40, 0x03, 0x10, 
  0x04, 0x00, 0x01, 0x20, 0xE4, 0x03, 0x00, 0x20, 0xE4, 0x03, 0x00, 0x20, 
  0xC4, 0x03, 0x00, 0x20, 0xC4, 0x00, 0x00, 0x20, 0x04, 0x00, 0x00, 0x20, 
  0xC8, 0x43, 0x00, 0x10, 0x08, 0x3F, 0x00, 0x10, 0x10, 0x00, 0x00, 0x08, 
  0x10, 0x00, 0x00, 0x08, 0x20, 0x00, 0x00, 0x04, 0xC0, 0x00, 0x00, 0x03, 
  0x00, 0x01, 0x80, 0x00, 0x00, 0x0E, 0x70, 0x00, 0x00, 0xF0, 0x0F, 0x00, 
  0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00,}},
//above is dog face
or:
 {"dogecoin",
{
  0x00, 0xF0, 0x0F, 0x00, 0x00, 0xFE, 0x7F, 0x00, 0x00, 0xFF, 0xFF, 0x01, 
  0xC0, 0xFF, 0xFF, 0x07, 0xE0, 0xFF, 0xFF, 0x0F, 0xF0, 0xFF, 0xFF, 0x0F, 
  0xF8, 0xFF, 0xFF, 0x1F, 0xF8, 0xFF, 0xFF, 0x3F, 0xFC, 0x0F, 0xF0, 0x7F, 
  0xFC, 0x0F, 0xC0, 0x7F, 0xFE, 0x8F, 0x81, 0x7F, 0xFE, 0xC7, 0x8F, 0xFF, 
  0xFE, 0xCF, 0x1F, 0xFF, 0xFE, 0xC7, 0x3F, 0xFF, 0xFF, 0xCF, 0x3F, 0xFF, 
  0xFE, 0x8B, 0x3E, 0xFE, 0xFE, 0x03, 0x3C, 0xFF, 0xFF, 0xC7, 0x3F, 0xFE, 
  0xFE, 0xCF, 0x3F, 0xFF, 0xFE, 0xCF, 0x1F, 0xFF, 0xFE, 0xCF, 0x1F, 0xFF, 
  0xFE, 0xCF, 0x87, 0x7F, 0xFE, 0x0F, 0x80, 0x7F, 0xFC, 0x07, 0xE0, 0x7F, 
  0xFC, 0x0F, 0xFC, 0x3F, 0xF8, 0xFF, 0xFF, 0x3F, 0xF0, 0xFF, 0xFF, 0x1F, 
  0xF0, 0xFF, 0xFF, 0x0F, 0xE0, 0xFF, 0xFF, 0x07, 0x80, 0xFF, 0xFF, 0x03, 
  0x00, 0xFF, 0xFF, 0x00, 0x00, 0xF8, 0x3F, 0x00, }};
//above is image of D coin

**Comment by eazee187** [comment]
Thanks for sharing!

How did you do that. Tried image2cpp and failed creating a code that worked. Could you please provide some hints?
Thanks a lot in advance!!!

**Reply by gruiny** [reply]
@eazee187  i found a solution 😅
first convert your pic to a ico 32x32 with this link Online:
http://image.convertstandard.com/toico32x32.aspx

After take the file and convert it again online on this site:
https://www.online-utility.org/image/convert/to/XBM

Open the file with a text editor and voila you have the code to insert in the "cryptos.h" file.

**Reply by eazee187** [reply]
@gruiny 

Oh man, thx a f...ing lot!!!

You're solution works beautiful!!! Greatly appreciated!!!

Now I'm on my search for good logos/pictures, I'm using cryptologos.cc.

Any hints? What are you using?
